internal/configs/static_evaluator_test.go

```diff
@@ -73,6 +73,9 @@ locals {
 	# Missing References
 	local_missing = local.missing
 	var_missing = var.missing
+
+	# Terraform 
+	ws = terraform.workspace
```

Comment:
You have added `ws` but did not add an assertion over it in the tests

Reply:
Good catch, added in 58023aebf5